java/src/main/java/org/whispersystems/signalservice/loki/api/LokiPushNotificationAcknowledgement.kt

```diff
@@ -0,0 +1,38 @@
+package org.whispersystems.signalservice.loki.api
+
+import okhttp3.*
+import org.whispersystems.libsignal.logging.Log
+import org.whispersystems.signalservice.internal.util.JsonUtil
+import java.io.IOException
+
+object LokiPushNotificationAcknowledgement {
+    //const val server = "https://live.apns.getsession.org/"
```

Comment:
Would be nice to automatically switch this based on the current build configuration so we don't run the risk of accidentally releasing with things pointing to the dev server